inspirehep/dojson/utils/__init__.py

```diff
@@ -224,3 +227,8 @@ def dedupe_all_lists(obj):
         return type(obj)(new_obj)
     else:
         return obj
+
+
+def validate(instance, schema):
+    return jsonschema_validate(
+        instance, schema, resolver=LocalRefResolver('', {}))
```

Comment:
@david-caro is 100% right, this method should live in `inspire_schemas.api`. But I have a conundrum: I'm improving the tests so that I can update `inspire_schemas` more easily, but this method will come in a future version of `inspire_schemas`, so I'm deadlocked!

To break the deadlock, I contract a little technical debt here by introducing this ugly little thing, with the intention of deleting it once the schemas are upgraded.